src/pages/Films/Films.tsx

```diff
@@ -1,30 +1,35 @@
 import { useEffect, useState } from "react";
-// import axios from "axios";
+import * as S from "./Films.Styled";
 import Carousel from "../../components/Carousel/Carousel";
-// import { GenreHeader } from "../../components/FilmGenreHeader/GenreHeader.Styled";
-// import Select from "../../components/Input/Select";
-import { FilmsStyled } from "./Films.Styled";
-import { GenreProps } from "../../helpers/types";
-// import Card from "../../components/Card/Card";
-// import image from "/the-dark-knight.jpg"
+import FilmsHeroImage from "../../components/FilmsBackground/FilmsBackground";
+import FilmsSearch from "../../components/FilmsSearch/FilmsSearch";
 
-function Films({ selectedGenre }: GenreProps) {
-	const apiKey = import.meta.env.VITE_API_KEY;
-	const URL: string = `http://www.omdbapi.com/?apikey=${apiKey}&s=Batman`;
+function Films() {
+	const [selectedGenre, setSelectedGenre] = useState(["Sci-Fi"]);
+	const [fetchedFilms, setFetchedFilms] = useState([]);
+	const [heroImage, setHeroImage] = useState<string>("");
 
-	const [films, setFilms] = useState([]);
+	const urlLeftSide = import.meta.env.VITE_URL_LEFT_SIDE;
+	const urlRightSide = import.meta.env.VITE_URL_RIGHT_SIDE;
+	const API_KEY = import.meta.env.VITE_API_KEY;
+
+	const URL = `${urlLeftSide + API_KEY + urlRightSide}`;
 
 	async function fetchData() {
 		try {
 			const res = await fetch(URL);
-			if (!res) {
-				console.error("Api data failure");
+			const data = await res.json();
+			console.log(data.results);
+			if (!data.results) {
```

Comment:
Nie zwraca to API jakiegoś obiektu np. data.error? albo coś? Spróbuj wysłać jakiś zły API KEY, i przetestuj jaki obiekt errora zwracają, może da się to jakoś wyświetlić w konsoli lub w notyfikacji systemowej.